### ISSUE TITLE
MD005 False positive when UL list is nested in OL and using standalone UL before

### ISSUE THREAD

**Issue by sumo-drosiek** [markdownlint/markdownlint#374]
## Description

Markdownlint reports error for given text, because first indentation for unordered list is 2 spaces based and items in second has to be 4 spaces based. Mdl returns no error if first list is removed

```markdown
- item 1.1
  - item 1.2

1. item 2:

    - item 2.1 # --> MD005 Inconsistent indentation for list items at the same level
    - item 2.2 # --> MD005 Inconsistent indentation for list items at the same level

```

### Environment

version: `0.11.0`

### Expected Behavior
No error is returned.

### Actual Behavior
Errors are in the description

## Replication Case
Create file with given content:
```
# header

- item 1.1
  - item 1.2

1. item 2:

    - item 2.1
    - item 2.2

```

run `mdl <path to file>`

```
$ mdl test.md 
test.md:8: MD005 Inconsistent indentation for list items at the same level
test.md:9: MD005 Inconsistent indentation for list items at the same level

A detailed description of the rules is available at https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md
```

**Comment by hoijui** [comment]
I get this one a lot, in many files/projects.

**Comment by lesinigo** [comment]
This is biting me too and seems possibly related to #296 ?

**Comment by markfilley** [comment]
I have the same issue. I run this a linter on my projects and tests require to pass. The nested numbering is only on a few documents. Going to do a sub list with `*` for now until this is updated.

**Comment by RochaStratovan** [comment]
This is still a problem. We're now just disabling the check. Which is sad because there are other instances where it is correctly finding problem.

**Comment by jaymzh** [comment]
This is fixed by #539
